### PR TITLE
SPT-939:KM - Point Core DID endpoint at bucket in SPOT account

### PR DESCRIPTION
**Pull request by chaitanguttarlapalli**
## Proposed changes
   -  import  spot key management bucket  for did generation
### What changed

  - Point the API Gateway endpoint in IPV Core at the bucket in the SPOT account.

   -This should rolled out in a Staging only (for the time-being).

### Why did it change

   -Monitoring of the bucket has been setup in SPOTs own account and, as the owner of the DID document, the document should really sit within our own boundaries.

### Issue tracking

- [SPT-939](https://govukverify.atlassian.net/jira/software/c/projects/SPT/boards/320?selectedIssue=SPT-939)

### Environment variables or secrets

- [x] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [x] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[SPT-939]: https://govukverify.atlassian.net/browse/SPT-939?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by chrisclayson — APPROVED** [review]
This looks good to me, @blakeyp should approve this before merging though 😄

**Review by blakeyp — APPROVED** [review]

**Pull request merged**
